repos/fdbt-site/src/pages/api/searchOperators.ts

```diff
@@ -34,7 +36,21 @@ export const isSearchInputValid = (searchText: string): boolean => {
     return searchRegex.test(searchText);
 };
 
-export default (req: NextApiRequestWithSession, res: NextApiResponse): void => {
+export const operatorsWithNoServers = async (selectedOperators: Operator[]): Promise<string[]> => {
```

Comment:
I think this needs to be renamed slightly, getOperatorsWithoutServices. Also it needs unit testing.